pkg/api/tests.go

```diff
@@ -249,3 +336,167 @@ func BuildTestsResults(dbc *db.DB, release, period string, collapse, includeOver
 
 	return testReports, overallTest, nil
 }
+
+func BuildTestsResultsFromBigQuery(bqc *bq.Client, release, period string, collapse, includeOverall bool, fil *filter.Filter) (testsAPIResultBQ, *apitype.TestBQ, error) { //lint:ignore
+	now := time.Now()
+
+	// Test results are generated by using two subqueries, which need to be filtered separately. Once during
+	// pre-processing where we're evaluating summed variant results, and in post-processing after we've
+	// assembled our final temporary table.
+	var rawFilter, processedFilter *filter.Filter
+	if fil != nil {
+		rawFilter, processedFilter = fil.Split([]string{"name", "variants"})
+	}
+
+	table := "junit_7day_comparison"
+	if period == "twoDay" {
+		table = "junit_2day_comparison"
+	}
+
+	dataSet := "openshift-gce-devel.ci_analysis_us"
```

Comment:
Can we use [BigQueryFlags](https://github.com/openshift/sippy/blob/main/pkg/flags/bigquery.go#L23) instead

Reply:
Thank you for the demo!  I think we just need to validate the dataset is infact `openshift-gce-devel.ci_analysis_us` since that is the only place we currently have the scheduled query in place.  I can create a separate card for the adhoc query support without the scheduled query that we discussed.